From 44e6f37a89081c11555b255b94e631b4724ed9ee Mon Sep 17 00:00:00 2001
From: henry shao <hshao@netlogicmicro.com>
Date: Wed, 30 Jun 2010 17:01:52 -0700
Subject: [PATCH 059/565] add node support for NMI_BASE

Add node support for NMI_BASE.

Based on Broadcom SDK 2.3.

Signed-off-by: henry shao <hshao@netlogicmicro.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/netlogic/xlp/cpu_control.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/mips/netlogic/xlp/cpu_control.c b/arch/mips/netlogic/xlp/cpu_control.c
index 8d1e1ac..8bc0464 100644
--- a/arch/mips/netlogic/xlp/cpu_control.c
+++ b/arch/mips/netlogic/xlp/cpu_control.c
@@ -53,7 +53,6 @@ THE POSSIBILITY OF SUCH DAMAGE.
 
 /* Globals Section 
  */ 
-#define XLP_ECFG_BASE		0x18000000
 #define XLP_SYS_DEV_BASE	0x35000
 uint32_t cores_bitmap = 0x1;	/* Bitmap of cores to enable  */
 int threads_to_enable; 			/* Threads to enable per core */
@@ -349,7 +348,7 @@ printk("[%s] node:%p \n", __FUNCTION__, node );
 		/* Bring the other cores online here
 	 	* Copy reset code into KSEG0 space
 	 	*/
-		if (cores_bitmap != 0xfe) {
+		if ((cores_bitmap != 0xfe)||(node > 0)) {
 			memcpy((void *)(NMI_BASE), 
 			   (void *)&reset_entry, 
 			   (reset_entry_end - reset_entry));
-- 
1.8.4.93.g57e4c17

